src/filters/FullPoseEstimator.h

```diff
@@ -3,9 +3,10 @@
 #include "../kinematics/DiffDriveKinematics.h"
 #include "../navtypes.h"
 #include "MultiSensorEKF.h"
+#include "../../src/navtypes.h"
 
 #include <array>
-
+using namespace navtypes;
```

Comment:
We should avoid using `using` directives in header files. Could you remove this?

As an aside, we should have a newline between the includes and the code too.